File: src/main/java/com/csl9015/gueaa/GameFor.java
package com.csl9015.gueaa;

import java.util.Random;
import java.util.Scanner;

public class GameFor {
    public static void main(String[] args) {
        int secret = new Random().nextInt(10);
        System.out.println(secret);
        Scanner scanner = new Scanner(System.in);
        int number = 0;

        for (int i = 1; i < 5; i++) {
            System.out.print("Please enter a number : (" + i + "/4)");
            number = scanner.nextInt();
            System.out.println("第" + i + "次 : " + number);
            if (number < secret) {
                System.out.println("higher");
            } else if (number > secret) {
                System.out.println("lower");
            } else {
                System.out.println("Great,the number is " + number);
                break;
            }
        }
    }
}

File: src/main/kotlin/com/csl9015/GameFor.kt
package com.csl9015

import java.util.*

fun main(args: Array<String>) {
    var secret = Random().nextInt(20) + 1
    println(secret)
    val scanner = Scanner(System.`in`)
    var number = 0

    for (i in 1..4) {
        print("Please enter a number ($i/4): ")
        number = scanner.nextInt()
        println("第${i}次$number")

        if (number < secret) {
            println("higher")
        } else if (number > secret) {
            println("lower")
        } else {
            println("Great,the number is $number")
            break
        }
    }
}